packages/dd-trace/test/plugins/tracing.spec.js

```diff
@@ -19,6 +22,7 @@ describe('TracingPlugin', () => {
           childOf: 'some parent span'
         })
       )
+      expect(span._spanContext._tags.component).to.equal('TracingPlugin')
```

Comment:
This is not a component.

Reply:
This assertion was removed and I reverted this test back to what it was like before. Since component is now getting tagged within the tracer.startSpan call along with the other tags, there is nothing to test here as the tracer.startSpan function is mocked in this test.